### DISCUSSION TITLE
Privacy Manifest @capacitor-firebase/authentication NSPrivacyAccessedAPICategoryFileTimestamp

### DISCUSSION THREAD

**Discussion by alexookah** [capawesome-team/capacitor-firebase#634] [General]
Hello,
I am getting this:
The Privacy Manifest file in your XCode project requires `NSPrivacyAccessedAPICategoryFileTimestamp` and a reason code for using it because @capacitor-firebase/authentication uses this feature.

[DOCS](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api#4278393)

What should be the correct option to use?

**Comment by robingenz** [comment]
That's strange, the plugin itself does not access any file timestamps. Feel free to take a look at the source code. Maybe the Firebase SDK does this.

**Comment by alexookah** [comment]
![Screenshot 2024-06-15 at 13 23 54](https://github.com/capawesome-team/capacitor-firebase/assets/9624843/fb475c36-754a-433d-97db-172e2ab1ab7d)

the ionic extension recommends this I dont think it comes from the firebase SDK, unless ionic runs pod install and checks pod dependencies privacy manifests.

**Reply by robingenz** [reply]
I suspect this is a bug in the Ionic VSCode extension.

**Reply by alexookah** [reply]
Ok I will try to remove it. If it's required AppStore should not allow me to upload if it's somehow used. right?

**Reply by robingenz** [reply]
Yes, see the [Apple announcement](https://developer.apple.com/news/?id=3d8a9yyh):

> Starting March 13: If you upload a new or updated app to App Store Connect that uses an API requiring approved reasons, we’ll send you an email letting you know if you’re missing reasons in your app’s privacy manifest. This is in addition to the existing notification in App Store Connect.

**Reply by alexookah** [reply]
It's not an ionic bug.
Firebase iOS SDK package had the issue as you suspected.

They fixed it in the firebase 10.25.0 SDK version
see relevant information here: 
https://github.com/firebase/firebase-ios-sdk/issues/12557#issuecomment-2120406485

@robingenz  Could we update the dependecy for iOS to the latest sdk 10.25.0 to resolve this issue for everyone?

**Reply by robingenz** [reply]
Done, see #642

**Reply by alexookah** [reply]
Very nice!
Thank you!

**Reply by alexookah** [reply]
When will you create a new capacitor firebase version?

**Reply by robingenz** [reply]
I will publish the next release in a few days.
